src/components/PeoplePage.tsx

```diff
@@ -15,15 +48,21 @@ export const PeoplePage = () => {
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
-
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+              {isLoading && <Loader />}
 
-              <p data-cy="noPeopleMessage">There are no people on the server</p>
+              {errorMessage && (
+                <p data-cy="peopleLoadingError">Something went wrong</p>
+              )}
 
-              <p>There are no people matching the current search criteria</p>
+              {people.length === 0 && !errorMessage && !isLoading && (
```

Comment:
```suggestion
              {!people.length && !errorMessage && !isLoading && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**